### ISSUE TITLE
xml are not decoded

### ISSUE THREAD

**Issue by kapodamy** [iBotPeaches/Apktool#2232]
### Information
v2.4.0
windows
google playstore/apkmirror.com

### Stacktrace/Logcat
none

### Steps to Reproduce
1. `java -Xmx1024m -jar apktool.jar d -o 6.66.05r13340-cc5db9b46-6660500  6.66.05r13340-cc5db9b46-6660500.apk`

### Frameworks
none

### APK
[6.66.05r13340-cc5db9b46-6660500.apk](
https://www.apkmirror.com/apk/9gag/9gag/9gag-6-66-05r13340-cc5db9b46-release/9gag-funny-gifs-pics-memes-videos-for-igtv-6-66-05r13340-cc5db9b46-android-apk-download/)

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? **no**
2. If you are trying to install a modified apk, did you resign it? **yes**
3. Are you using the latest apktool version? **yes**

tl;dr
lastest version doesnt decode any xml on the apk (including the manifest).
but the version 2.3.4 works correctly

**Comment by kapodamy** [comment]
nvm is apk studio issue
